lib/rack/constants.rb

```diff
@@ -52,6 +52,7 @@ module Rack
   RACK_RECURSIVE_INCLUDE              = 'rack.recursive.include'
   RACK_MULTIPART_BUFFER_SIZE          = 'rack.multipart.buffer_size'
   RACK_MULTIPART_TEMPFILE_FACTORY     = 'rack.multipart.tempfile_factory'
+  RACK_RESPONSE_FINISHED              = 'rack.response_finished'
```

Comment:
@BlakeWilliams @tenderlove looking at other keys, should this be `rack.response.finished` or `rack.response_finished`? We don't use `rack.request_form_input` but `rack.request.form_input`.

Is there any precend for appending `_callbacks`? I don't mind either way, just calling it out.